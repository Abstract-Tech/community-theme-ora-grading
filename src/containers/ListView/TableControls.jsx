import React from 'react';

import {
  DataTable,
} from '@edx/paragon';

/**
 * <TableControls />
 */
export const TableControls = () => (
  <>
    <DataTable.TableControlBar />
    <DataTable.Table />
    <DataTable.EmptyTable content="No results found" />
    <DataTable.TableFooter />
  </>
);
TableControls.propTypes = {};

export default TableControls;
