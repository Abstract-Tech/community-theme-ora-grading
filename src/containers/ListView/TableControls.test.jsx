import React from 'react';
import { shallow } from 'enzyme';

import TableControls from './TableControls';

jest.mock('@edx/paragon', () => ({
  DataTable: {
    TableControlBar: () => 'DataTable.TableControlBar',
    Table: () => 'DataTable.Table',
    EmptyTable: () => 'DataTable.EmptyTable',
    TableFooter: () => 'DataTable.TableFooter',
  },
}));

describe('ListView TableControls component', () => {
  describe('component', () => {
    test('snapshot', () => {
      expect(shallow(<TableControls />)).toMatchSnapshot();
    });
  });
});
