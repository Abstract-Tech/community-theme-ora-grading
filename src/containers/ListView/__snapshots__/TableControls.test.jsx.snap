// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListView TableControls component component snapshot 1`] = `
<Fragment>
  <TableControlBar />
  <Table />
  <EmptyTable
    content="No results found"
  />
  <TableFooter />
</Fragment>
`;
